m4_sessions/1903_inning/automation.md

```diff
@@ -1,53 +1,48 @@
 ## Group Eight's Session Plan
 
-M4 Student Leaders: Mike, Justin, Jesse, Ethan, Dan
+M4 Student Leaders: Justin, Leah, Matthew, Nick 
 
-# Turing Career Development: Automation
+# Turing Career Development: Automation: Job Elimination 101
 
 ## Session Outcomes
 
-## Mission Alignment		 
-  		  
-*Our mission is to unlock human potential by training a diverse, inclusive student body to succeed in high-fulfillment technical careers.
-  		  
-As your technical careers progress, most of the work you will do is to automate processes that have been done manually. Our mission is to “unlock human potential.” The software we write enables it’s users to be more productive, and to direct energy toward solving increasingly difficult problems. However, some automation, especially of physical processes, can force some out of the jobs they’ve done for years. This has the potential to do more harm than good for inclusion in our society.
+As your technical careers progress, most of the work you will do is to automate processes that have been done manually. Our mission at Turing is to “unlock human potential.” The software we write enables it’s users to be more productive, and to direct energy toward solving increasingly difficult problems. However, some automation, especially of physical processes, can force some out of the jobs they’ve done for years. This has the potential to do more harm than good for inclusion in our society.
 
-## Pre-Session
+#### Introductions 1:00 - 1:05
+  * Have students introduce themselves
+    Name:
+    CoHort: 
+    Pronouns:
+    Answer Questions:
+   1. If you could have one process in your life automated tomorrow what would it be?
+   2. What automated process today do you appreciate the most?
 
+#### Icebreaker 1:05 - 1:10
+   Automation icebreaker?
 
-### Survey Questions
-Survey to be sent either prior to session or immediately after ( prior may lead to better response? )
+### Groupings 1:10 - 1:15
+  We will group members by opposing stances. Have students stand in a spectrum of how much they like the idea of automation, and from that hand select one each from opposing sides and a middle. This should facilitate the most conversation and opposing viewpoints.
```

Comment:
Will it be groups of 3/4?

Reply:
Hopefully four groups.  Size will depend on number of attendees.